airflow/cli/commands/dag_command.py

```diff
@@ -125,16 +124,15 @@ def dag_delete(args):
     :return:
     """
     api_client = get_current_api_client()
-    log = LoggingMixin().log
     if args.yes or input(
             "This will drop all existing records related to the specified DAG. "
             "Proceed? (y/n)").upper() == "Y":
         try:
             message = api_client.delete_dag(dag_id=args.dag_id)
+            print(message)
         except OSError as err:
-            log.error(err)
+            print(str(err))
             raise AirflowException(err)
```

Comment:
Do we need print here? The same exception with message will be rise.

Reply:
Fixed. Raising exceptions is also not the best solution, but it is another problem.